Permission Error

Here's the report @avegrv

![photo_2021-01-11_19-39-54](https://user-images.githubusercontent.com/219005/104183608-cf751c00-5444-11eb-9c66-d11cbfbe7535.jpg)


`2021-01-11 19:35:48.376 32053-32053/com.sample.google.drive E/MainActivity: error upload file
    com.google.api.client.googleapis.extensions.android.gms.auth.UserRecoverableAuthIOException
        at com.google.api.client.googleapis.extensions.android.gms.auth.GoogleAccountCredential$RequestHandler.intercept(GoogleAccountCredential.java:297)
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:868)
        at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:476)
        at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:409)
        at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.execute(AbstractGoogleClientRequest.java:526)
        at com.sample.google.drive.google.GoogleDriveApiDataRepository.lambda$createFile$5$GoogleDriveApiDataRepository(GoogleDriveApiDataRepository.java:124)
        at com.sample.google.drive.google.-$$Lambda$GoogleDriveApiDataRepository$3pIxHFJqPM9iMVYFazgjDLChDzg.call(lambda)
        at com.google.android.gms.tasks.zzv.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
        at java.lang.Thread.run(Thread.java:761)
     Caused by: com.google.android.gms.auth.UserRecoverableAuthException: NeedPermission
        at com.google.android.gms.auth.zze.zzb(Unknown Source)
        at com.google.android.gms.auth.zzd.zza(Unknown Source)
        at com.google.android.gms.auth.zzd.zzb(Unknown Source)
        at com.google.android.gms.auth.zzd.getToken(Unknown Source)
        at com.google.android.gms.auth.zzd.getToken(Unknown Source)
        at com.google.android.gms.auth.zzd.getToken(Unknown Source)
        at com.google.android.gms.auth.GoogleAuthUtil.getToken(Unknown Source)
        at com.google.api.client.googleapis.extensions.android.gms.auth.GoogleAccountCredential.getToken(GoogleAccountCredential.java:267)
        at com.google.api.client.googleapis.extensions.android.gms.auth.GoogleAccountCredential$RequestHandler.intercept(GoogleAccountCredential.java:292)
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:868) 
        at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:476) 
        at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:409) 
        at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.execute(AbstractGoogleClientRequest.java:526) 
        at com.sample.google.drive.google.GoogleDriveApiDataRepository.lambda$createFile$5$GoogleDriveApiDataRepository(GoogleDriveApiDataRepository.java:124) 
        at com.sample.google.drive.google.-$$Lambda$GoogleDriveApiDataRepository$3pIxHFJqPM9iMVYFazgjDLChDzg.call(lambda) 
        at com.google.android.gms.tasks.zzv.run(Unknown Source) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607) 
        at java.lang.Thread.run(Thread.java:761) 
`

_Originally posted by @Stillhard in https://github.com/avegrv/google-drive-api-sample/issues/1#issuecomment-757926581_

--- a/app/src/main/java/com/sample/google/drive/google/GoogleDriveActivity.java
+++ b/app/src/main/java/com/sample/google/drive/google/GoogleDriveActivity.java
@@ -46,7 +46,6 @@ public abstract class GoogleDriveActivity extends GoogleSignInActivity {
 
     private void initializeDriveClient(GoogleSignInAccount signInAccount) {
         List<String> scopes = new ArrayList<>();
-        scopes.add(DriveScopes.DRIVE_FILE);
         scopes.add(DriveScopes.DRIVE_APPDATA);
 
         GoogleAccountCredential credential = GoogleAccountCredential.usingOAuth2(this, scopes);

